server/statistics/util.go

```diff
@@ -34,3 +34,10 @@ const (
 func storeTag(id uint64) string {
 	return fmt.Sprintf("store-%d", id)
 }
+
+func hotStatReportInterval(kind RWType) int {
```

Comment:
how about move this function to the RWtype sub function like this: `func(kind RWType) interval ()int{}`